.github/semantic.yml

```diff
@@ -0,0 +1,29 @@
+allowMergeCommits: true
+# Always validate the PR title AND all the commits
+titleAndCommits: true
+types:
+    - feat
+    - fix
+    - docs
+    - style
+    - refactor
+    - perf
+    - test
+    - build
+    - ci
+    - revert
+scopes:
```

Comment:
How about also having `Common` or `Internal` for the common code in https://github.com/edgexfoundry/edgex-go/tree/master/internal/pkg